site/index.css

```diff
@@ -43,28 +43,57 @@ body {
   float: right;
   color: #FFFAFB;
   font-family: "Palanquin Dark";
+  font-size: 1.1em;
   /* todo: choose alt fonts */
 }
 
 #aquapants-quote {
+  margin-top: 1.5em;
   margin-bottom: 1.5em;
```

Comment:
Let's just add the margin to all sides of the quote block so that it looks symmetrical.

We can remove 

```suggestion
  margin: 1.5em;
```

![image](https://github.com/aquapants/website/assets/123187978/0e027bca-72be-4b31-a03f-bc02383da811)